Makefile

```
@@ -5,7 +5,7 @@
 TOP := $(dir $(firstword $(MAKEFILE_LIST)))
 
 # Variables we update as newer versions are released
-BOTTLEROCKET_SDK_VERSION = v0.28.0
+BOTTLEROCKET_SDK_VERSION = v0.30.1
```

```suggestion
BOTTLEROCKET_SDK_VERSION ?= v0.30.1
```
Perhaps in a second commit, could we switch from `=` to `?=` for **BOTTLEROCKET_SDK_VERSION** and **BUILDER_IMAGE**? Then we could easily test changes locally with `BUILDER_IMAGE=public.ecr.aws/c3k7p7u2/thar-be-a-beta-sdk-x86_64:v0.30.1 make controller`